website/content/docs/enterprise/audit/exclusion.mdx

```diff
@@ -0,0 +1,285 @@
+---
+layout: docs
+page_title: Exclusion syntax for audit results
+description: >-
+  Learn about the behavior and syntax for excluding audit data in Vault Enterprise.
+---
+
+# Exclusion syntax for audit results
+
+@include 'alerts/enterprise-only.mdx'
+
+As of Vault 1.18.0, you can enable audit devices with an `exclude` option to exclude
+specific fields in an audit entry that is written to a particular audit log, and fine-tune
+your auditing process.
+
+<Warning title="Proceed with caution">
+
+  Excluding audit entry fields is an advanced feature. Use of exclusion settings
+  could lead to missing data in your audit logs.
+
+  **Always** test your audit configuration in a non-production environment
+  before deploying exclusions to production. Read the
+  [Vault security model](/vault/docs/internals/security) and
+  [filtering overview](/vault/docs/concepts/filtering) to familiarize yourself
+  with Vault auditing and filtering basics before enabling audit devices that use
+  exclusions.
+
+</Warning>
+
+Once you enable an audit device with exclusions, every audit entry Vault sends to
```

Comment:
`every audit entry Vault` might take paraphrasing a little bit for easy reading.